### PR TITLE
Display "No filters available" message for a user who has not logged in or has not filters available (#526).

### PR DESCRIPTION
**Pull request by nimishamukherjee**
Display "No filters available" message for a user who has not logged in or has not filters available (#526).

### CODE REVIEW

File: src/app/work-item/work-item-list/work-item-list.component.html [hunk 1]
```diff
@@ -8,9 +8,13 @@ <h4 class="list-type">Backlog</h4>
         <span class="caret"></span>
       </button>
       <ul class="dropdown-menu dropdown-menu-right" role="menu" aria-labelledby="wi_filter_dropdown">
-        <li *ngFor="let filter of filters;" role="presentation" (click)="activeFilter(filter.id)">
+        <li *ngIf="filters.length == 0">          
+          <a>No filters available</a>
```
Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-list.component.html]: this should come in span because <a> tag will come with its other stylings too

Comment (sanbornsen) [inline src/app/work-item/work-item-list/work-item-list.component.html]: I think the anchor tag is fine here. We want all the styles of it, don't we?

Comment (SMahil) [inline src/app/work-item/work-item-list/work-item-list.component.html]: Do we need clickable No filters?

Comment (nimishamukherjee) [inline src/app/work-item/work-item-list/work-item-list.component.html]: I used the <a> tag to retain the styling.

**Pull request merged**